e2e/docker/docker.go

```diff
@@ -40,6 +40,7 @@ type Provider struct {
 	servicesOnce sync.Once
 	testnet      types.Testnet
 	omniTag      string
+	explorerTag  string
```

Comment:
can just use `omniTag`

Reply:
We wanted to deploy this yesterday but `main` branch was broken. Therefore when deploying we decided to go with a different tag so that we could deploy older version of everything else and a newer version of the explorer. We'll remove the separate tag today.